services/gcn_service/gcn_service.py

```diff
@@ -12,26 +12,15 @@
 init_db(**cfg['database'])
 
 
-@gcn.include_notice_types(
-    gcn.NoticeType.FERMI_GBM_FLT_POS,
-    gcn.NoticeType.FERMI_GBM_GND_POS,
-    gcn.NoticeType.FERMI_GBM_FIN_POS,
-    gcn.NoticeType.FERMI_GBM_SUBTHRESH,
-    gcn.NoticeType.LVC_PRELIMINARY,
-    gcn.NoticeType.LVC_INITIAL,
-    gcn.NoticeType.LVC_UPDATE,
-    gcn.NoticeType.LVC_RETRACTION,
-    # gcn.NoticeType.LVC_TEST,
-    gcn.NoticeType.AMON_ICECUBE_COINC,
-    gcn.NoticeType.AMON_ICECUBE_HESE,
-    gcn.NoticeType.ICECUBE_ASTROTRACK_GOLD,
-    gcn.NoticeType.ICECUBE_ASTROTRACK_BRONZE,
-)
 def handle(payload, root):
-
-    user_id = 1
-    with DBSession() as session:
-        post_gcnevent(payload, user_id, session)
+    notice_type = gcn.get_notice_type(root)
+    if notice_type in [
```

Comment:
Maybe this list should be read in at the same time as the config instead of again for every new event?

Reply:
I'm not so sure what you mean by that. Want me to save that list once in a variable that then i use in this if statement ?

Reply:
Yeah that's all I meant.

Reply:
I'll take care of it soon then !

Reply:
Done